test/benchee/formatters/tagged_save_test.exs

```diff
@@ -1,12 +1,15 @@
 defmodule Benchee.Formatters.TaggedSaveTest do
   use ExUnit.Case
 
-  alias Benchee.{Suite, Statistics}
-  alias Benchee.Benchmark.Scenario
-  alias Benchee.Formatter
-  alias Benchee.Formatters.TaggedSave
+  alias Benchee.{
+    Benchmark.Scenario,
+    Formatter,
+    Formatters.TaggedSave,
+    Statistics,
+    Suite
+  }
 
-  import Benchee.Formatters.TaggedSave
+  import TaggedSave
```

Comment:
Not sure what credo says but I'd rather not alias `TaggedSave` and rather import it with the fully qualified name